### PR TITLE
Research detail page

### PR DESCRIPTION
**Pull request by KalobTaulien**
Ticket #8387

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html [hunk 1]
```diff
@@ -2,67 +2,122 @@
...
+  <div class="tw-bg-no-repeat tw-bg-center tw-bg-cover tw-min-h-[144px]" style='background-image: url("{{ hero_img.url }}")'></div>
+
+
+  <div class="tw-max-w-6xl tw-mx-auto tw-pb-[100px] tw--mt-7 tw-bg-white tw-px-5 tw-pt-5">
+    <div class="medium:tw-grid tw-grid-cols-5 tw-grid-rows-1 tw-grid-flow-col tw-gap-7">
+      <div class="tw-col-span-1 tw-pt-7">
+        {# TODO Add image #}
+        TODO: Add image
+        {% comment %}
+        {% image page.image fill-259x344 as img %}
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html:17]: Probably best done with a method on the page to retrieve the parent (library) banner image.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html [hunk 2]
```diff
@@ -2,67 +2,122 @@
...
+                    {% endfor %}
+                  </div>
+                {% endif %}
+                {% if page.related_topics %}
+                  <div class="large:tw-px-4 large:tw-py-0 tw-pt-4">
+                    {% for related_topic in page.related_topics.all  %}
+                      <span>{{ related_topic.research_topic.name }}{% if not forloop.last %}, {% endif %}</span>
+                    {% endfor %}
+                  </div>
+                {% endif %}
```
[the remark this answers is not in the document]
Comment (KalobTaulien) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html:54]: Ah I see that now. We need the author_image in the template too (for the research authors). But Ill apply this to the related topics section right away.

[the remark this answers is not in the document]
Comment (tbrlpld) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html:54]: Yea, I see. My comment was mostly meant for the metadata section below the heading.

**Comment by KalobTaulien** [comment]
@tbrlpld Fixed up everything you found to the exception of the `research_authors` because we need the author images (if they one have one) and the author name. Wasn't sure if you were working on that or if you'd mind if I stepped in and added that, so I left it as is for now.

**Review by tbrlpld — CHANGES_REQUESTED** [review]
@KalobTaulien thanks for working on this. 

There are a few lines that need consideration, partly because of overlapping work. 

A few other points that are not code comments that I noticed while running your branch locally: 

- [ ] On mobile, the other of cover image and the heading section is inverted. The heading should come above the cover image. 
- [ ] The little header image nook on the right is missing on mobile and tablet sizes
- [ ] The metadata style does not seem to match the design. I get some strange border on the left and top. You can probably have a look at the detail card fragment if you are interested in a reference implementation of very similar styling. 
- [ ] The author names should link to the respective author detail page. I have updated the acceptance criteria to make this explicit. 
- [ ] The tablet layout had been added to the designs. We should update the implementation to reflect that. 
- [ ] On desktop, the heading section should be only as wide as the "overview" section. Currently it takes the whole width of the cut out.

**Comment by tbrlpld** [comment]
@KalobTaulien Don't worry about the nook in the banner image. I am moving this to the base template, because it's the same on all research hub pages except the landing page.

**Comment by tbrlpld** [comment]
Closing this in favour of #8730 8730

**Pull request closed without merging**